src/components/TopBar/OhmMenu.jsx

```diff
@@ -108,7 +108,7 @@ function OhmMenu() {
     //               </Link>
     //
     <Link
-      href={`https://app.uniswap.org/#/swap?inputCurrency=${daiAddress}&outputCurrency=${GURU_ADDRESS}`}
+      href={`https://app.sushi.com//#/swap?inputCurrency=${daiAddress}&outputCurrency=${GURU_ADDRESS}`}
```

Comment:
don't think the # should be in there
https://app.sushi.com/swap?inputCurrency=0xFdAD081F3dD9A97A9F935DD14050D4F8807E2644&outputCurrency=0xfC8F2F8a8929E23473Be54c027b9Aa03452a5348

Reply:
fixed